data-raw/crops_obic/b_lu_brp.csv

```diff
@@ -0,0 +1,461 @@
+B_LU_BRP,B_LU_NAME,B_REGION,B_LU,B_LU_WATERSTRESS_OBIC,B_LU_CROPROTATIONINTENSITY_OBIC,B_LU_EOM,B_LU_EOM_RESIDUE,B_LU_HC,B_LU_CULTCAT4,B_LU_CROPROTATION,B_LU_CRUMBLEABILITY_OBIC,B_LU_PCLASS_OBIC,B_LU_SEALINGCLASS,B_LU_NEVALCLASS_OBIC,B_LU_KEVALCLASS_OBIC,B_LU_MEASURE_OBIC,B_LU_BOTANICAL,B_LU_SEASON,B_LU_MAKKINK,B_LU_BBWP,B_LU_ARABLE_ER,B_LU_PRODUCTIVE_ER,B_LU_CULTIVATED_ER,B_LU_ROOTBALL_DIAMETER,B_LU_PLANT_DENSITY
```

Comment:
Can the number of columns be reduced by removing not relevant columns, like _er?

Reply:
These are removed from the final product, `crops.obic` when preparing `crops.obic` in "data-rwa/crops_obic/crops_obic.R". It is my understanding that data in "data-raw" is not distributed.
 This csv file is sourced from `pandex::b_lu_brp`. In my opinion, the responsibility of trimming the eventual data should lay with the data using package, not the data providing package.

Reply:
Ok, then we can remove this file as it sourced from pandex?

Reply:
We will prepare a subset of b_lu_brp in pandex including the rows and columns required by OBIC to be readable in OBIC